internal/cli/atlas/atlas.go

```diff
@@ -102,3 +110,14 @@ func Builder() *cobra.Command {
 
 	return cmd
 }
+
+func newDeprecatedMessage() string {
+	if runtime.GOOS != "windows" {
+		deprecatedMessage := `There’s a new, dedicated Atlas CLI available for Atlas users. Install the Atlas CLI to enjoy the same capabilities and keep getting new features. Run "brew install mongodb-atlas-cli" or visit https://dochub.mongodb.org/core/migrate-to-atlas-cli.
```

Comment:
`brew install` is not common for linux systems

Reply:
also please consider using build tags instead of a runtime `if`, see https://github.com/mongodb/mongodb-atlas-cli/tree/master/internal/mongosh

Reply:
I will open a follow-up PR to inject the GOOS at build time instead of runtime. I don't want to make to many changes here as we have the doc files that don't help to review this PR

Reply:
Hide the `.txt` files, that's how I've been reviewing this 
![Screenshot 2022-03-28 at 12 23 41](https://user-images.githubusercontent.com/461027/160387924-d13c8fd1-b5e9-421a-aa86-759ec2bf03f8.png)

Reply:
ack. I will make the change here. Thanks